host-configs/tioga-toss_4_x86_64_ib_cray-clang@17.0.0_hip.cmake

```diff
@@ -0,0 +1,146 @@
+
+set(MPI_Fortran_COMPILER "/usr/tce/packages/cray-mpich-tce/cray-mpich-8.1.27-rocmcc-5.7.1/bin/mpif90" CACHE PATH "")
+
+set(MPIEXEC_NUMPROC_FLAG "-n" CACHE STRING "")
+
+set(ENABLE_MPI ON CACHE BOOL "")
+
+set(MPIEXEC_EXECUTABLE "/usr/global/tools/flux_wrappers/bin/srun" CACHE PATH "")
+
+#------------------------------------------------------------------------------
+# Hardware
+#------------------------------------------------------------------------------
+
+#------------------------------------------------
+# ROCm
+#------------------------------------------------
+
+set(HIP_ROOT_DIR "/opt/rocm-5.7.1/hip" CACHE PATH "")
+
+set(CMAKE_HIP_COMPILER "/opt/rocm-5.7.1/llvm/bin/clang++" CACHE FILEPATH "")
+
+set(CMAKE_HIP_ARCHITECTURES "gfx90a" CACHE STRING "")
+
+set(AMDGPU_TARGETS "gfx90a" CACHE STRING "")
+
+set(GPU_TARGETS "gfx90a" CACHE STRING "")
+
+set(ENABLE_OPENMP OFF CACHE BOOL "")
+
+set(ENABLE_HIP ON CACHE BOOL "")
+
+set(HIP_CLANG_INCLUDE_PATH "/opt/rocm-5.7.1/hip/../llvm/lib/clang/17.0.0/include" CACHE PATH "")
+
+set(CMAKE_EXE_LINKER_FLAGS "-L/opt/rocm-5.7.1/hip/../llvm/lib -L/opt/rocm-5.7.1/hip/lib -Wl,-rpath,/opt/rocm-5.7.1/hip/../llvm/lib:/opt/rocm-5.7.1/hip/lib -lpgmath -lflang -lflangrti -lompstub -lamdhip64 -L/opt/rocm-5.7.1/hip/../lib64 -Wl,-rpath,/opt/rocm-5.7.1/hip/../lib64 -L/opt/rocm-5.7.1/hip/../lib -Wl,-rpath,/opt/rocm-5.7.1/hip/../lib -lamd_comgr -lhsa-runtime64 -lhipblas -lrocblas -lrocsolver " CACHE STRING "")
+
+#------------------------------------------------------------------------------
+# TPLs
+#------------------------------------------------------------------------------
+
+set(TPL_ROOT "/usr/WS2/smithdev/libs/serac/toss_4_x86_64_ib_cray/2024_10_01_15_45_47/clang-17.0.0" CACHE PATH "")
+
+set(AXOM_DIR "${TPL_ROOT}/axom-0.9.0.1-e3mw22b6kayepwgdt56bigytlodcj67y" CACHE PATH "")
+
+set(CAMP_DIR "${TPL_ROOT}/camp-2024.02.0-s5oevzf32vll33gkf3lqj2w5ihuptosv" CACHE PATH "")
+
+set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.9.1-waf3ywzaectbjklqzubsx5w2ijakqpc7" CACHE PATH "")
+
+set(LUA_DIR "/usr" CACHE PATH "")
+
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.7.0.1-c2zdyi4i5oplimxje3zqujjcqo2zmqqi" CACHE PATH "")
+
+set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.23-2xc27hk7hdalhfw5dzhn5etitbmmpv3g" CACHE PATH "")
+
+set(HYPRE_DIR "${TPL_ROOT}/hypre-2.26.0-qtj2nrzsgakqcjbkfsaqsajqa526uq6a" CACHE PATH "")
+
+set(METIS_DIR "${TPL_ROOT}/metis-5.1.0-7xwxuepf642wq3q676sews7trkep5lna" CACHE PATH "")
+
+set(PARMETIS_DIR "${TPL_ROOT}/parmetis-4.0.3-l326pd32hmhmvcvhi7oijukafts5ukyz" CACHE PATH "")
+
+set(NETCDF_DIR "${TPL_ROOT}/netcdf-c-4.7.4-vjh33ikfewgb5f52gisy4vxnuiai4jgo" CACHE PATH "")
+
+set(SUPERLUDIST_DIR "${TPL_ROOT}/superlu-dist-8.1.2-jo6kcgx33o2jdwyzl5jksv2cly5ne4xz" CACHE PATH "")
+
+set(ADIAK_DIR "${TPL_ROOT}/adiak-0.4.0-pmhmyqnlwzuth7gxhpkdymcfj254lqmx" CACHE PATH "")
+
+# AMGX not built
+
+set(CALIPER_DIR "${TPL_ROOT}/caliper-2.10.0-5yg5mgygsul5op4jgtxypp7jvtb5ojcc" CACHE PATH "")
+
+# PETSC not built
```

Comment:
Please make an issue to track we need to eventually turn back on PETSC for HIP

Reply:
https://github.com/LLNL/serac/issues/1243